javascript/ql/lib/semmle/javascript/security/dataflow/XssThroughDomCustomizations.qll

```diff
@@ -216,4 +216,33 @@ module XssThroughDom {
       }
     }
   }
+
+
+  /**
```

Comment:
```suggestion

  /**
```
There is a newline too much here, and I think that's what the autoformatter is complaining about this time.